src/Parsers/ASTCreateQuery.h

```diff
@@ -101,17 +103,15 @@ class ASTCreateQuery : public ASTQueryWithTableAndOutput, public ASTQueryWithOnC
     bool has_uuid{false}; // CREATE TABLE x UUID '...'
 
     ASTColumns * columns_list = nullptr;
-
-    StorageID to_table_id = StorageID::createEmpty();   /// For CREATE MATERIALIZED VIEW mv TO table.
-    UUID to_inner_uuid = UUIDHelpers::Nil;      /// For materialized view with inner table
-    ASTStorage * inner_storage = nullptr;      /// For window view with inner table
```

Comment:
Here in this PR I've moved these three fields `to_table_id`, `to_inner_uuid`, `inner_storage` (and `storage` when it's used to specify the inner storage for a materialized view or window views) to new class `ASTViewTargets`. This is because for TimeSeries tables I need three more types of target tables so I thought it could be better to keep all view targets together (because they're handled in the same way sometimes).